### PR TITLE
Added unit test related to the issue #5114

### PR DESCRIPTION
**Pull request by rchigvintsev**
I've just written a simple unit test that can find bugs similar to that one described in the issue #5114. Should I create an issue on https://issues.jboss.org too?

### CODE REVIEW

File: jcache/embedded/src/test/java/org/infinispan/jcache/JCacheLoaderAdapterTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.infinispan.marshall.core.MarshalledEntryFactory;
+import org.infinispan.marshall.core.MarshalledEntryFactoryImpl;
+import org.infinispan.persistence.DummyInitializationContext;
+import org.infinispan.persistence.spi.InitializationContext;
+import org.infinispan.test.AbstractInfinispanTest;
+import org.infinispan.util.DefaultTimeService;
+import org.infinispan.util.TimeService;
+import org.testng.annotations.*;
+
+import static org.testng.AssertJUnit.*;
```
Comment (ryanemerson) [inline jcache/embedded/src/test/java/org/infinispan/jcache/JCacheLoaderAdapterTest.java]: Can you replace the `*` imports please. At the moment these imports cause checkstyle to fail

**Comment by ryanemerson** [comment]
@rchigvintsev Overall looks good to me, just the minor changes to the import statements are required

**Comment by wburns** [comment]
@rchigvintsev the issue you created for the original one is fine https://issues.jboss.org/browse/ISPN-7783 I can update the commit and add the PR to it.

**Comment by wburns** [comment]
Integrated into master, thanks @rchigvintsev

**Comment by rchigvintsev** [comment]
@wburns OK. Then close this issue.

**Comment by wburns** [comment]
Just misclicked 😀 meant to close the issue.

**Pull request closed without merging**